### DISCUSSION TITLE
Artado Search hakkında

### DISCUSSION THREAD

**Discussion by islemci** [GuvendeKal/guvendekal.org#66] [Oylama]
Konu hakkındaki BTT Community üzerindeki tartışma: https://btt.community/t/gizlilik-rehberi-guvendekal-org/12584/26

> Artado Search, kullanıcılara arama deneyimlerini kendi benzersiz ihtiyaçlarına göre uyarlama olanağı sağlamak üzere tasarlanmış, çok yönlü ve son derece özelleştirilebilir bir arama motorudur. Bu proje ASP.NET Framework'ü temel alır ve AGPL v3 lisansı altında gururla açık kaynaktır. Yalnızca kendi arama sonuçlarını sunmakla kalmaz, aynı zamanda diğer arama motorlarından gelen sonuçları da entegre ederek kapsamlı bir arama çözümü sunar. ([Artado Search anasayfası](https://artadosearch.com))

Sizin fikirlerinizi merak ediyoruz. Lütfen aşağıdan oyunuzu kullanın! 👇

**Comment by GokturkTalha** [comment]
Benim şahsi fikrim projenin daha eklenmek için erken olması.

**Comment by conmecha** [comment]
Bilinen arama motorları kadar olmasa da biraz daha olgunlaşması daha doğru olacaktır. Ayrıyetten [gizlilik potikalarına](https://www.artadosearch.com/privacy) bakılacak olursa tam anlamıyla bir gizlilik sağlamıyorlar. Performansı yeterli değil (ki bu anlaşılabilir bir şey, henüz yeterli bütçe yok fakat sonuçta yetersiz). Diğer rakiplere kıyasla da daha az özelliğe sahip. Şu an için fazla erken.
